site/gatsby-site/src/templates/report.js

```diff
@@ -24,8 +24,8 @@ function ReportPage(props) {
   const { loading, isRole } = useUserContext();
 
   if (report.language !== locale) {
-    report.title = data[locale].title;
-    report.text = data[locale].text;
+    report.title = data[locale]?.title || '';
```

Comment:
I found a better way to fix this lack of translations for the report. Please take a look at this [PR](https://github.com/responsible-ai-collaborative/aiid/pull/2562/files)
In that way, we keep the default `title` and `text` instead of set to empty string.